google/ads/google_ads/v2/types.py

```diff
@@ -1637,7 +1635,7 @@
 
 def _get_class_from_module(module_name):
     module = importlib.import_module(module_name)
-    for class_name, _ in getmembers(module, isclass): # from inspect module
+    for class_name, _ in get_messages(module).items(): # from inspect module
```

Comment:
The key line in get_messages is: 
```
if inspect.isclass(candidate) and issubclass(candidate, message.Message):
```
We can avoid another import if we just use that. 
Ref: https://github.com/googleapis/google-cloud-python/blob/dfffcc7c0a9208e00065378c6a59167a817c51f7/api_core/google/api_core/protobuf_helpers.py#L116

Reply:
I'd prefer to use the helper method as-is since we already use it elsewhere in the file, and in the future it's possible the rules for how these classes are filtered could change.

Reply:
Thinking about it a bit more, I think we should go with my proposed solution, otherwise each module may be imported twice, leading to potential performance parity issues.

Reply:
How can the modules be imported twice? I'm not seeing how that would happen.

Reply:
ok, let's proceed with `get_messages()`